.travis.yml

```diff
@@ -8,7 +8,7 @@ install:
 
 script:
 - npm run lint
-- npm run lerna:bootstrap
+- npm run setup
```

Comment:
Should we use `setup:ci` here?

Reply:
I was working on removing `--no-package-lock` from Lerna configuration, but I could not tested it correctly and I did not push the lock files. So, the `setup:ci` should be removed instead.